### DISCUSSION TITLE
Easy integration of editing a MarkupString

### DISCUSSION THREAD

**Discussion by NeoCoderMatrix86** [Megabit/Blazorise#3899] [Q&A]
Hello, 

I currently use a "`<Text>`" component for displaying a MarkupString (containing "`<Mark>Test</Mark>`") to the user. Now I would like to have the user edit the String (No HTML edit) and have an event, when it was edited for recognizing changes and running recognition again. Is there some component I can use for this purpoise?

**Comment by stsrki** [comment]
Not sure exactly what you mean. But I don't think there is an alternative for `MarkupString`. We also use it https://github.com/Megabit/Blazorise/blob/master/Demos/Blazorise.Demo/Pages/Tests/MarkdownPage.razor#L18

We also use the Markdig library to convert from markup string into HTML, which is then used with MarkupString.
